### PR TITLE
Fix UT failure config_test and context_test

### PR DESCRIPTION
**Pull request by gzmkl**
Disable two unit tests in config_test.py and context_test.py
because  get_memory_usage() and get_memory_usage() do not throw exception for CPU device in case of MKL is enabled. 

More details: 

1. For default cpu allocator (framework/cpu_allocator_impl.cc), the global Boolean variable  cpu_allocator_collect_stats is set to true by direct_session by invoking EnableCPUAllocatorStats(). 

2. However, for MKL cpu allocator (common_runtime/mkl_cpu_allocator.h/.cc), its static Boolean variable mkl_small_allocator_collect_stats cannot be set to true, as the MKL cpu allocator cannot re-define the method EnableCPUAllocatorStats(). 

TODO: work with Google team to address the limitation in framework/allocator.h, or find a workaround.

### CODE REVIEW

**Comment by gzmkl** [comment]
will reopen after internal check

**Comment by gzmkl** [comment]
Reopen after pinning down true root cause. This PR fixes the two test cases for now.

File: tensorflow/python/eager/context_test.py [hunk 1]
```diff
@@ -136,8 +136,12 @@ def testGetMemoryInfo(self):
 
   @test_util.disable_tfrt('b/169293680: TFE_GetTotalMemoryUsage is unsupported')
   def testGetMemoryInfoCPU(self):
-    with self.assertRaisesRegex(ValueError, 'Allocator stats not available'):
-      context.context().get_memory_info('CPU:0')
+    # MklCPUAllocator (mkl_cpu_allocator.h) does not throw exception.
+    # So skip the test.
+    # TODO(gzmkl) work with Google team to address design issue in allocator.h
+    if not test_util.IsMklEnabled():
```
Comment (kkimdev) [inline tensorflow/python/eager/context_test.py:142]: nit:

```
if test_util.IsMklEnabled():
  # TODO(gzmkl): work with Google team to address design issue in allocator.h
  self.skipTest("MklCPUAllocator (mkl_cpu_allocator.h) does not throw exception. So skip the test.")
```

Please update other parts as well.

**Comment by penpornk** [comment]
@gzmkl Gentle ping. The TF 2.6 branch cut is this Thursday at 5pm PT.

File: tensorflow/python/eager/context_test.py (hunk 1 above)
Comment (gzmkl) [reply tensorflow/python/eager/context_test.py]: @kkimdev Thank you for the suggestion, based on which I have made code change

**Review by kkimdev — APPROVED** [review]

**Pull request merged**